lib/index.js

```diff
@@ -18,6 +18,17 @@ var render = require('posthtml-render')
  * @param {Array} plugins - An array of PostHTML plugins
  */
 function PostHTML (plugins) {
+  /**
+ * PostHTML Instance
+ *
+ * @name posthtml
+ * @version 0.9.1
+ *
+ * @prop plugins
+ * @prop options
+ */
+  this.version = '0.9.1'
```

Comment:
@voischev the first interpolation. i have create new issue when will configured build process with babili, babel and there will be to write a plugin to replace the version.